types/dependency-cruiser.d.ts

```diff
@@ -441,7 +879,7 @@ export interface IReporterOutput {
  *                        [here](../src/cli/flattenTypeScriptConfig.js))
  */
 export function cruise(
-  pFileDirArray: string[],
+  pFileDirArray: ICruiseResult[],

```

Comment:
This should stay a `string[]` - it's the files 'n folders the consumer puts in. You might've confused this parameter with the one in the `format` function below :-)

Reply:
Okay, I will get it fixed